src/stores/chatExtras.js

```diff
@@ -61,6 +64,19 @@ export const useChatExtrasStore = defineStore('chatExtras', {
 		getNextEvent: (state) => (token) => {
 			return state.upcomingEvents[token]?.[0]
 		},
+
+		getChatSummaryTaskQueue: (state) => (token) => {
+			return Object.values(Object(state.chatSummary[token]))
+		},
```

Comment:
```suggestion
		},
		
		hasTaskFailed: (state) => (token) => {
			return Object.values(Object(state.chatSummary[token])).any(task => task.status === TASK_PROCESSING.FAILED)
		},
```

Use this to show the hint that sthg went wrong in summarizing. If you think that there are many hints (along with AI disclaimer), then don't show the summary, stop any tasks after the failed ones and show the failed summary text.

Reply:
Follow-up